x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/RestEsqlTestCase.java

```diff
@@ -1014,12 +1014,14 @@ public void testMultipleBatchesWithLookupJoin() throws IOException {
         // Create more than 10 indices to trigger multiple batches of data node execution.
         // The sort field should be missing on some indices to reproduce NullPointerException caused by duplicated items in layout
         for (int i = 1; i <= 20; i++) {
-            createIndex("idx" + i, randomBoolean(), "\"mappings\": {\"properties\" : {\"a\" : {\"type\" : \"keyword\"}}}");
+            createIndex("no_sort_field_idx" + i, randomBoolean(), "\"mappings\": {\"properties\" : {\"a\" : {\"type\" : \"keyword\"}}}");
```

Comment:
I wonder if we should go one step forward and rely on `randomIdentifier()` for the index name here?